### PR TITLE
cleanup Pathfinder#goto when Pathfinder#stop is called

### PR DESCRIPTION
**Pull request by GoogleSites**
Calling `Pathfinder#stop` while a path from `Pathfinder#goto` is being travelled causes it to stop indefinitely, making it throw an error when `Pathfinder#goto` is called again.

I've added the `path_stop` listener to `Pathfinder#goto`, which will clean up when called while the path is still being used.

### CODE REVIEW

**Comment by Karang** [comment]
ci-test doesn't passes, run `npm run fix`

File: lib/goto.js [hunk 1]
```diff
@@ -51,6 +56,7 @@ function goto (bot, goal) {
       }, 0)
     }
 
+    bot.on('path_stop', pathStopped);
```
Comment (u9g) [inline lib/goto.js:59]:
```suggestion
    bot.on('path_stop', pathStopped)
```

**Comment by rom1504** [comment]
looks ok to me, what do you think @Karang ?

**Comment by IceTank** [comment]
LGTM

**Pull request merged**